pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -25,25 +32,17 @@ let
     "LD_LIBRARY_PATH=${cudaStub}\${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH} ";
 
 in buildPythonPackage rec {
-  version = "1.0.0";
+  version = "1.1.0";
   pname = "pytorch";
 
   src = fetchFromGitHub {
     owner  = "pytorch";
     repo   = "pytorch";
     rev    = "v${version}";
     fetchSubmodules = true;
-    sha256 = "076cpbig4sywn9vv674c0xdg832sdrd5pk1d0725pjkm436kpvlm";
+    sha256 = "1rckv7m3h04mgf7p61rmabszgxh5110ly6qq0qnp28vg7mckvgkh";
   };
 
-  patches =
```

Comment:
I believe this patch can be removed since it was patching with a v1.1.0 commit. I can verify on a single-gpu instance later.